config/migrations/2023/subsidies/20231116154454-add-opvolgmoment-2024-step.sparql

```diff
@@ -0,0 +1,44 @@
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX subsidie: <http://data.vlaanderen.be/ns/subsidie#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX qb: <http://purl.org/linked-data/cube#>
+PREFIX cpsv: <http://purl.org/vocab/cpsv#>
+PREFIX xkos: <http://rdf-vocabulary.ddialliance.org/xkos#>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    # add follows reference to the subsidy measure offer
+    <http://lblod.data.info/id/subsidy-measure-offers/64d40351-8128-464f-990f-41066154583e> cpsv:follows <http://data.lblod.info/id/subsidy-procedural-steps/ecccdad1-69c4-45d1-ad42-61ba4d168f22> .
+
+    # add link from subsidy-measure-offer-series to subsidy-procedural-step
+    <http://lblod.data.info/id/subsidy-measure-offer-series/51d5ede8-4e7a-4d80-ab1b-2bdc5294fdcf> lblodSubsidie:heeftSubsidieprocedurestap <http://data.lblod.info/id/subsidy-procedural-steps/ecccdad1-69c4-45d1-ad42-61ba4d168f22>.
+
+    <http://lblod.data.info/id/subsidie-application-flow-steps/629edada-bea3-418d-be40-9a5b1ba8caf9> xkos:next <http://lblod.data.info/id/subsidie-application-flow-steps/3514ae35-e18e-48b5-8647-80c06d926c78>.
+
+
+    # create new subsidie-application-flow-step
+    <http://lblod.data.info/id/subsidie-application-flow-steps/3514ae35-e18e-48b5-8647-80c06d926c78> a lblodSubsidie:ApplicationStep;
+      mu:uuid "3514ae35-e18e-48b5-8647-80c06d926c78";
+      qb:order 3;
+      xkos:previous  <http://lblod.data.info/id/subsidie-application-flow-steps/629edada-bea3-418d-be40-9a5b1ba8caf9>;
+      dct:references <http://data.lblod.info/id/subsidy-procedural-steps/ecccdad1-69c4-45d1-ad42-61ba4d168f22> ;
+      dct:isPartOf <http://lblod.data.info/id/subsidie-application-flows/049c2a16-ba71-42c7-82d4-0749c3ccf9c2>; #this should be the application-flow of climate
+      dct:source <config://forms/climate/oproep-2/step-submit-opvolgmoment-2024/versions/20231116154454/form.ttl>.
+
+    # Create new subsidy-procedural-step
+    <http://data.lblod.info/id/subsidy-procedural-steps/ecccdad1-69c4-45d1-ad42-61ba4d168f22> a subsidie:Subsidieprocedurestap;
+      mu:uuid "ecccdad1-69c4-45d1-ad42-61ba4d168f22";
+      dct:description """Opvolgmoment 2024""";
+      mobiliteit:periode <http://data.lblod.info/id/periodes/cea1f5f8-a659-4f59-886a-fe8f75cb2305>.
+
+    # New period for step opvolgmoment 2024
+    <http://data.lblod.info/id/periodes/cea1f5f8-a659-4f59-886a-fe8f75cb2305> a <http://data.europa.eu/m8g/PeriodOfTime>;
+      mu:uuid "cea1f5f8-a659-4f59-886a-fe8f75cb2305";
+      # Time is set to 07 december 23:00 because UTC+1 this becomes 08 december 00:00 local time
+      <http://data.europa.eu/m8g/startTime> "2023-12-07T23:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>;
+      # Time is set to 01 march 21:59:00 because Belgium time is UTC+2 instead of UTC+1 in March and it becomes 23:59.
+      <http://data.europa.eu/m8g/endTime> "2024-03-01T21:59:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>.
```

Comment:
This needs to be `"2024-03-01T22:59:00Z"` because the Belgian timezone would still be `UTC+1h` on 01 March, 2024; otherwise, the subsidy will close prematurely. The timezone turns into `UTC+2h` on 31 March, 2024 ( https://www.timeanddate.com/time/change/belgium/brussels?year=2024 ).

Reply:
Ahh you are correct, good catch